src/main/java/seedu/task/model/ModelManager.java

```diff
@@ -255,7 +255,7 @@ public String toString() {
         public boolean run(ReadOnlyTask task) {
             return taskKeyWords.stream()
                     .filter(keyword -> StringUtil.containsIgnoreCase(task.getTask().fullName, keyword) 
-                    		|| StringUtil.containsIgnoreCase(task.getDescription().value, keyword))
+                    		|| StringUtil.containsIgnoreCase(task.getDescription().orElse(null).toString() , keyword))
```

Comment:
Yeah, thats a great idea. Forgot I had that method. Edited to getDescriptionValue()